packages/ironic/charts/templates/deployment.yaml

```diff
@@ -257,11 +257,3 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- with .Values.global.dnsPolicy }}
-      dnsPolicy:
-        {{- toYaml . | nindent 8 }}
```

Comment:
I think we need to explicitly set `dnsPolicy: ClusterFirstWithHostNet` because this pod runs with `hostNetwork: true`